server/scoreboard.c

```diff
@@ -704,3 +722,231 @@ AP_DECLARE(global_score *) ap_get_scoreboard_global()
+                    ready++;
+                }
+                else if (res != SERVER_DEAD &&
+                         res != SERVER_STARTING && res != SERVER_IDLE_KILL &&
+                         ps->generation == mpm_generation) {
+                    busy++;
+                } else {   
+                    dead++;
+                }   
+            }
+
+            if (ap_extended_status && !ps->quiescing && ps->pid) {
+                if (ws->access_count != 0 
+                    || (res != SERVER_READY && res != SERVER_DEAD)) {
+#ifdef HAVE_TIMES
+                    tmp_tu = ws->times.tms_utime;
+                    tmp_ts = ws->times.tms_stime;
+                    tmp_tcu = ws->times.tms_cutime;
+                    tmp_tcs = ws->times.tms_cstime;
+
+                    if (times_per_thread) {
+                        proc_tu += tmp_tu;
+                        proc_ts += tmp_ts;
+                        proc_tcu += tmp_tcu;
+                        proc_tcs += tmp_tcs;
+                    }
+                    else {
+                        if (tmp_tu > proc_tu ||
+                            tmp_ts > proc_ts ||
+                            tmp_tcu > proc_tcu ||
+                            tmp_tcs > proc_tcs) {
+                            proc_tu = tmp_tu;
+                            proc_ts = tmp_ts;
+                            proc_tcu = tmp_tcu;
+                            proc_tcs = tmp_tcs;
+                        }
+                    }
+#endif /* HAVE_TIMES */
+                    sl->access_count += ws->access_count;
+                    sl->bytes_served += ws->bytes_served;
+                    sl->duration += ws->duration;
+                }
+            }
+        }
+#ifdef HAVE_TIMES
+        sl->cpu_usr += proc_tu + proc_tcu;
+        sl->cpu_sys += proc_ts + proc_tcs;
+#endif
+    }
+#ifdef HAVE_TIMES
+    global_record = ap_get_scoreboard_global();
+    sl->cpu_usr += global_record->times.tms_utime
+                   + global_record->times.tms_cutime;
+    sl->cpu_sys += global_record->times.tms_stime
+                   + global_record->times.tms_cstime;
+#endif
+    total = busy + ready + dead;
+    if (total) {
+        sl->idle = (float)ready * 100 / total;
+        sl->busy = (float)busy * 100 / total;
+        sl->dead = (float)dead * 100 / total;
+    }
+}
+
+static void calc_mon_data(ap_mon_snap_t *last,
+                          ap_mon_snap_t *next)
+{
+    unsigned long accesses;
+    ap_sload_t *s0 = last->sload;
+    ap_sload_t *s1 = next->sload;
+#ifdef HAVE_TIMES
+    float tick;
+#endif
+
+    next->acc_per_sec = -1;
+    next->bytes_per_sec = -1;
+    next->average_concurrency = -1;
+    next->cpu_load = -1;
+    next->bytes_per_acc = -1;
+    next->ms_per_acc = -1;
+    next->interval = -1;
+
+    /* Need two iterations for complete data in s0 and s1 */
+    if (s0->access_count < 0 || s0->access_count < 0) {
```

Comment:
One of the two should be `s1` probably.

Reply:
Thanks, fixed in https://github.com/apache/httpd/pull/116/commits/946e7b4939b21e5d8823c18e20f59e934424dabb